### DISCUSSION TITLE
Access to shared GPU cublas/cublasLt/cudnn library handles from custom_op kernels

### DISCUSSION THREAD

**Discussion by renderist** [microsoft/onnxruntime#15286] [Other Q&A]
The GPU custom_op examples only shows direct CUDA programming examples, where the CUDA stream handle is accessible via the API. The provider and contrib_ops show access to cublas, cublasLt, and cudnn NVidia library handles. How might one gain access to these library handles from a custom_op?

**Comment by pranavsharma** [comment]
Can you file this as an issue? This is not the first time this has been asked. https://github.com/microsoft/onnxruntime/issues/14284. Added to our backlog.

**Comment by renderist** [comment]
Thanks Pranav,

I added my comments on the feature request that you pointed out to me, is there any utility to filing it as a new issue in terms of escalation?

Thanks,

Kevin


On Mar 30, 2023, at 2:37 PM, Pranav Sharma ***@***.******@***.***>> wrote:



Can you file this as an issue? This is not the first time this has been asked. #14284<https://github.com/microsoft/onnxruntime/issues/14284>. Added to our backlog.

**Reply by pranavsharma** [reply]
Just easier to measure the demand :-).

**Comment by renderist** [comment]
Understood.

Issue #15296


On Mar 30, 2023, at 9:32 PM, Pranav Sharma ***@***.******@***.***>> wrote:



Just easier to measure the demand :-).
